pkg/drivers/kvm/network.go

```diff
@@ -478,9 +478,19 @@ func ifListFromAPI(conn *libvirt.Connect, domain string) ([]libvirt.DomainInterf
 	}
 	defer func() { _ = dom.Free() }()
 
-	ifs, err := dom.ListAllInterfaceAddresses(libvirt.DOMAIN_INTERFACE_ADDRESSES_SRC_LEASE)
-	if err != nil {
-		return nil, fmt.Errorf("failed listing network interface addresses of domain %s: %w", domain, err)
+	ifs, err := dom.ListAllInterfaceAddresses(libvirt.DOMAIN_INTERFACE_ADDRESSES_SRC_ARP)
+	if ifs == nil {
+		if err != nil {
+			log.Debugf("failed listing network interace addresses of domain %s(source=arp): %w", domain, err)
```

Comment:
log.Debugf("failed listing network interace addresses of domain %s(source=arp): %w", domain, err)
```suggestion
			log.Debugf("failed listing network interface addresses of domain %s(source=arp): %w", domain, err)
```

Reply:
Ah..
..and fixed now.